<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QDialog" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_credit_amount">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_interest_rate">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest rate(%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_credit_period">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit period(month)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="amount_cr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>9999999999999.000000000000000</double>
   </property>
   <property name="value">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="period_cr">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>10000000000000000.000000000000000</double>
   </property>
   <property name="value">
    <double>12.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rate_interest">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>18.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_total_payment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>215</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>9999999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_overpayment">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>215</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>9999999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_payment">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> Total payment </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_overpayment">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_month_payment">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Month payment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate_credit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate credit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>189</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_ann">
      <property name="text">
       <string>Ann</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_diff">
      <property name="text">
       <string>Diff</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit_month_payment">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>215</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
